template/scripts/drupal-vm/drupal-vm.drushrc.aliases.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+// [vagrant_machine_name].local
+$aliases['${project.machine_name}.local'] = array(
+  // /var/www/[vagrant_machine_name]/docroot
+  'root' => '/var/www/${project.machine_name}/docroot',
+  // vagrant_hostname
+  'uri' => '${project.local_uri}',
+  // vagrant_hostname
+  'remote-host' => '${project.local_uri}',
+  'remote-user' => 'vagrant',
+  'ssh-options' => '-i ~/.vagrant.d/insecure_private_key',
```

Comment:
this doesn't work for `core-rsync` see https://github.com/geerlingguy/drupal-vm/pull/552.